cypress/support/cypress-support/src/utils.js

```diff
@@ -1288,6 +1288,33 @@ function determineActionFromFeatureFile(testRunnable) {
   return 'CORE';
 }
 
+/**
+ * Builds intent environment and resolves intent using Cypress config module.
+ * @param {string} appId - The durable app ID.
+ * @param {string} programType - The type of program (e.g., 'movie', 'episode').
+ * @returns {Chainable<any>} Cypress chainable for resolved intent.
+ */
+function buildAndResolveIntent(appId, programType) {
+  const runtimeEnv = Cypress.env();
```

Comment:
We shouldn't need to do any of this. The env variables that are required should be able to be fetched in the configMod.